application/src/main/java/org/togetherjava/tjbot/commands/tags/TagsCommand.java

```diff
@@ -45,24 +44,25 @@ public TagsCommand(TagSystem tagSystem) {
 
     @Override
     public void onSlashCommand(@NotNull SlashCommandEvent event) {
+        int MAX_TAGS_THRESHOLD_WARNING = 200;
```

Comment:
Constants are usually `private static final` fields in the class. Id suggest you change that. Otherwise, looks good.